packages/core/webpack/published-ts-components.js

```diff
@@ -39,8 +39,8 @@ const publishedTSComponents = {
   Slider: "components/Slider/Slider",
   BreadcrumbsBar: "components/BreadcrumbsBar/BreadcrumbsBar",
   BreadcrumbItem: "components/BreadcrumbsBar/BreadcrumbItem/BreadcrumbItem",
-  ColorPicker: "components/ColorPicker/ColorPicker",
-  ColorPickerContent: "components/ColorPicker/components/ColorPickerContent/ColorPickerContent",
+  ColorPicker: "components/ColorPickerExperimental/ColorPickerExperimental",
+  ColorPickerContent: "components/ColorPickerExperimental/components/ColorPickerContent/ColorPickerContent",
```

Comment:
This shouldn't be changed as it break the existing ColorPicker, you only need the exports in next.ts